static/js/settings_org.js

```diff
@@ -22,15 +22,14 @@ exports.maybe_disable_widgets = function () {
     $(".organization-box [data-name='auth-methods']")
         .find("input, button, select, checked")
         .prop("disabled", true);
-
+    $("#deactivate_realm_button").attr("disabled", true);
     if (page_params.is_admin) {
```

Comment:
I think that this is not the correct fix for this. There was this recent commit 498fe285fa7bc750af573883c1c80b1963cf91e4 in which we changed disabled to be used as property instead of attribute. I think the correct fix here is to replace `[disabled="disabled"]` with just `[disabled]` in `app_components.css` as we are using disabled as property and not attribute.

Reply:
ok @sahil839  thanks for the review